.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -468,13 +470,28 @@ def write(x=''):
             }
         }
 
+        if args.worker_pair:
+            worker_dict["submitty-worker"] = {
+                "capabilities": ['default'],
+                "address": "172.18.2.8",
+                "username": "submitty",
+                "num_autograding_workers": NUM_GRADING_SCHEDULER_WORKERS,
+                "enabled": True
+            }
+
         if args.setup_for_sample_courses:
             worker_dict['primary']['capabilities'].extend([
                 'cpp',
                 'python',
                 'et-cetera',
                 'notebook',
             ])
+            worker_dict['submitty-worker']['capabilities'].extend([
+                'cpp',
+                'python',
+                'et-cetera',
+                'notebook',
+            ])
```

Comment:
Shouldn't this be under the `if args.worker_pair` branch? Why would we want to set the capabilities for the machine if it doesn't exist?